File: Backend/Events/Converters/Parties/Digimons/DigievolutionConverter.cs
using Backend.Domain.Models.Parties.Digimons;
using Backend.Events.DTO.Parties.Digimons;

namespace Backend.Events.Converters.Parties.Digimons;

public static class DigievolutionConverter
{
    public static DigievolutionDTO ToDTO(Digievolution digievolution) => new()
    {
        Level = digievolution.Level
    };
}

File: Frontend/src/events/converters/digievolution.converter.ts
import type { DigievolutionDTO } from '../dto/parties/digimons/digievolution.dto';
import type { Digievolution } from '../../models';

export class DigievolutionConverter {
    public static convert(digievolutionDto: DigievolutionDTO): Digievolution {
        return {
            level: digievolutionDto.level
        };
    }
}
